src/fragments/lib/logging/common/change_log_levels/30_category_log_level.mdx

```diff
@@ -0,0 +1,22 @@
+Add a `categoryLogLevel` section and specify each category and its log level.
```

Comment:
can we add a list of the categories available below the json blob please?

Reply:
add a section at the bottom to list out the existing categories after the code snippet.